h_program-lang/skip_code.mli

```diff
@@ -8,13 +8,28 @@ type skip =
 
 val load: Common.filename -> skip list
 
+(*
+   Return selected paths (unchanged) and excluded files (real paths including
+   the root).
+*)
 val filter_files:
-  skip list -> Common.dirname (* root *) -> Common.filename list ->
-  Common.filename list
+  skip list -> root:Common.dirname (* root *) -> Common.filename list ->
+  Common.filename list * Common.filename list
+
+(*
+   TODO: explain why 'root' is a list and the implementation only uses its
```

Comment:
Isn't it better to explain it now?

Reply:
I'm not the author and I haven't figured this out. This is for @aryx.